packages/react/src/date-picker/docs/overview.mdx

```diff
@@ -1,49 +1,47 @@
 ---
 title: Date picker
-description: 'Date picker allows users to select a single date or range of dates. There are 3 types: date select range, simple date input and date select.'
+description: The Date picker component allows users to select a single date via a calendar or text input.
```

Comment:
Do we have some kind of guidance to say `Date picker` instead of `DatePicker` when talking about the component? This always feels weird to me.

Reply:
The convention is to use sentence case (e.g. `Date picker`) and this recommendation came from the content team. 

This component documentation (and many others) was written before the recommendation though.